R/lambda.inverse.sim.R

```diff
@@ -3,9 +3,8 @@
 ## Copyright (c) Felix Andrews <felix@nfrac.org>
 ##
 
-
-## TODO: keep attributes of Q
```

Comment:
Keep this TODO if this wasn't a reason to remove it?

Reply:
No problem, it's been added back.